Source/WebCore/page/EventHandler.cpp

```diff
@@ -486,6 +486,11 @@ void EventHandler::nodeWillBeRemoved(Node& nodeToBeRemoved)
 {
     if (nodeToBeRemoved.containsIncludingShadowDOM(RefPtr { m_clickNode }.get()))
         m_clickNode = nullptr;
+
+    if (nodeToBeRemoved.containsIncludingShadowDOM(m_lastElementUnderMouse.get())) {
+        m_lastElementUnderMouse = nullptr;
+        m_lastScrollbarUnderMouse = nullptr;
```

Comment:
Are we sure m_lastScrollbarUnderMouse is always inside m_lastElementUnderMouse?

Reply:
No, I'm not sure. How do we check it? My intention  is to remove  m_lastElementUnderMouse but I noticed that at https://searchfox.org/wubkat/source/Source/WebCore/page/EventHandler.cpp#2805, m_lastScrollbarUnderMouse was removed at the same time. Maybe I should leave m_lastScrollbarUnderMouse  out since I'm not sure?

Reply:
Leaving that out for now seems like a good move.

Reply:
Thanks! Code updated.